receiver/datadogreceiver/receiver.go

```diff
@@ -109,3 +133,87 @@ func (ddr *datadogReceiver) handleTraces(w http.ResponseWriter, req *http.Reques
 	_, _ = w.Write([]byte("OK"))
 
 }
+
+// handleV1Series handles the v1 series endpoint https://docs.datadoghq.com/api/latest/metrics/#submit-metrics
+func (ddr *datadogReceiver) handleV1Series(w http.ResponseWriter, req *http.Request) {
```

Comment:
Perhaps not for this PR, but it might be a good idea to split the handlers in their own source file.

Reply:
Do you mean something like `metrics_receiver.go` and `traces_receiver.go`?